app/src/routes/onboarding/forms/user.ts

```diff
@@ -0,0 +1,254 @@
+import { Field } from '@directus/types';
+import { useI18n } from 'vue-i18n';
+
+export const useUserFields = function (): Field[] {
+	const { t } = useI18n();
+	return [
+		{
+			collection: 'onboarding',
+			name: t('fields.directus_users.first_name'),
+			field: 'first_name',
+			type: 'string',
+			schema: {
+				name: 'first_name',
+				table: 'onboarding',
+				schema: 'public',
+				data_type: 'character varying',
+				is_nullable: false,
+				generation_expression: null,
+				default_value: null,
+				is_generated: false,
+				max_length: 255,
```

Comment:
```suggestion
				max_length: 50,
```
The database has a max length of 50 here